dev_apps/music-nga/test/marionette/lib/music.js

```diff
@@ -0,0 +1,493 @@
+/* global require, module */
+'use strict';
+
+var assert = require('assert');
+
+function Music(client, origin) {
+  this.client = client;
+  this.origin = origin || ('app://' + Music.DEFAULT_ORIGIN);
+  this.actions = client.loader.getActions();
+}
+
+module.exports = Music;
+
+Music.DEFAULT_ORIGIN = 'music-nga.gaiamobile.org';
+
+Music.Selector = Object.freeze({
+  viewFrame: '#view-stack iframe',
```

Comment:
I'm not sure in what all context this selector is used, but if you want to ensure you're getting the currently-visible iframe, you can use `'iframe.active'`